frontend/src/fun/jp-ristet/jp-ristet.mjs

```diff
@@ -159,6 +159,10 @@ class JPRistetElement extends LitElement {
       background-color: var(--jp-color-bg-4);
     }
 
+    .grid-cell.path {
+      background-color: #999;
```

Comment:
Better to use one of the `--jp-color-bg-x` variables, which are gonna change with light vs dark mode